contracts/interfaces/ISynchronizer.sol

```diff
@@ -5,66 +5,65 @@ pragma solidity ^0.8.11;
 import "./IMuonV02.sol";
 
 interface ISynchronizer {
-
-    event Buy(address partnerID, address user, address registrar, uint256 deiAmount, uint256 price, uint256 collateralAmount, uint256 feeAmount);
-    event Sell(address partnerID, address user, address registrar, uint256 registrarAmount, uint256 price, uint256 collateralAmount, uint256 feeAmount);
+    event Buy(address partnerId, address receipient, address registrar, uint256 amountIn, uint256 price, uint256 collateralAmount, uint256 feeAmount);
+    event Sell(address partnerId, address receipient, address registrar, uint256 amountIn, uint256 price, uint256 collateralAmount, uint256 feeAmount);
     event WithdrawFee(address platform, uint256 partnerFee, uint256 platformFee, uint256 registrarType);
-    event MinimumRequiredSignatureSet(uint256 oldValue, uint256 newValue);
-    event AppIdSet(uint8 oldID, uint8 newID);
-    event VirtualReserveSet(uint256 oldReserve, uint256 newReserve);
-    event MuonContractSet(address oldContract, address newContract);
+    event SetMinimumRequiredSignatures(uint256 oldValue, uint256 newValue);
```

Comment:
You transferred every "Set" from the end of the event names to the beginning.
So we need to either make this format `UseVirtualReserveToggled` same this `SetMuonContract` or undo their names to the old format like `MuonContractSet`.

Reply:
What are you proposing for the UseVirtualReserveToggled exactly? Something like "SetUseVirtualReserve" ?

Reply:
It doesn't set, It toggles the `useVirtualReserve` variable, But we can use `SetUseVirtualReserve` too.

Reply:
Exactly. Because it toggles and not SETs I decided to leave it as-is. Or would you prefer `ToggleUseVirtualReserve` then?

Reply:
Yes, It seems good. Please use `ToggleUseVirtualReserve`